README.md

```diff
@@ -122,6 +122,82 @@ With [django-csp](https://django-csp.readthedocs.io/en/latest/nonce.html#middlew
 {% block script_attributes %}nonce="{{request.csp_nonce}}"{% endblock %}
 ```
 
+## ASGI support
+
+Async views with an ASGI server is also supported.
+
+```python
+import asyncio
+
+from suspense.shortcuts import async_render
+
+# app/views.py
+async def view(request):
+    async def obj():
+
+        await asyncio.sleep(1)
+        return range(10)
+
+    return async_render(request, 'template.html', {'obj': obj()})
```

Comment:
What do you think about using a single entrypoint to handle page rendering? I mean to use only one and the same `render` function. This way the user can simultaneously work with synchronous and asynchronous objects in one request (corner case). Open for discussion.

Reply:
ASGI with streaming response needs all conditions to be set to work properly:
- `async` view
- asgi server
- Streaming response with an `async` streaming function.

if anything is not ok it will run like a regular synchronous `HttpResponse`.

`render` could instead have an  "async" kwargs. but the user still have to make sure to have an async view with an ASGI server.

The Async Template class view still might be necessary.